backend/src/modules/schedules/schedules.module.ts

```diff
@@ -15,7 +19,7 @@ import { createSchedulesService, deleteSchedulesService } from './schedules.prov
 		forwardRef(() => BoardsModule),
 		CommunicationModule
 	],
-	providers: [createSchedulesService, deleteSchedulesService, boardRepository],
+	providers: [createSchedulesService, deleteSchedulesService, boardRepository, scheduleRepository],
```

Comment:
```suggestion
	providers: [createSchedulesService, deleteSchedulesService, scheduleRepository],
```
I think the board repository isn't needed